rust/gsc/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "gsc"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+entity = { path = "../entity" }
+migration = { path = "../migration" }
+sea-orm = { version = "0.11.3", features = ["sqlx-postgres", "runtime-tokio-native-tls"]}
+serde = "1.0.164"
+tokio = { version = "1.28.2", features = ["full"] }
+# tower-http = "0.4.1"
```

Comment:
Commented code

Reply:
I intentionally left this here as we're likely to use this in the future. I can get rid of it now if you'd like however.

Reply:
removed